src/main/java/com/cognifide/aemrules/utils/Throwables.java

```diff
@@ -0,0 +1,50 @@
+/*-
+ * #%L
+ * AEM Rules for SonarQube
+ * %%
+ * Copyright (C) 2015-2019 Cognifide Limited
```

Comment:
Same as `Comparables`. Just erring on the side of caution :)

Reply:
copied from Guava 
https://guava.dev/releases/23.0/api/docs/com/google/common/base/Throwables.html

/ apache license

https://github.com/google/guava/blob/master/COPYING